statistics/update_test.go

```diff
@@ -890,7 +886,7 @@ func (s *testStatsUpdateSuite) TestLogDetailedInfo(c *C) {
 	for i := 0; i < 20; i++ {
 		testKit.MustExec(fmt.Sprintf("insert into t values (%d, %d, %d)", i, i, i))
 	}
-	testKit.MustExec("analyze table t")
+	testKit.MustExec("analyze table t limit 4 buckets")
```

Comment:
Why use 4 buckets here?

Reply:
It is the test's requirement. Each bucket should contain more than 1 values.